A_arithmetic_progression.py

```diff
@@ -0,0 +1,25 @@
+from data_reader import DataReader
+data_reader = DataReader("sums_in_loop.txt")
+
+data = data_reader.get_table_int(" ")
+print(data)
+
+#first value opens the sequence
```

Comment:
FYI - we should not commit commented code. If it is not necessary to run your program - simple remove it

Reply:
Done.